### PR TITLE
Automatically deploy PR to the Expo (for easier reviews)

### PR DESCRIPTION
**Pull request by mrtnzlml**

### CODE REVIEW

**Comment by mrtnzlml** [comment]
Please test these changes in [Expo](https://docs.expo.io/versions/latest/introduction/installation.html#mobile-client-expo-for-ios-and-android):

  QR Code

**Comment by mrtnzlml** [comment]
Please test these changes in [Expo](https://docs.expo.io/versions/latest/introduction/installation.html#mobile-client-expo-for-ios-and-android):

  QR Code

**Comment by mrtnzlml** [comment]
Please test these changes in [Expo](https://docs.expo.io/versions/latest/introduction/installation.html#mobile-client-expo-for-ios-and-android):

  QR Code

**Comment by mrtnzlml** [comment]
Please test these changes in [Expo](https://docs.expo.io/versions/latest/introduction/installation.html#mobile-client-expo-for-ios-and-android):

  QR Code

File: scripts/expo/commands/publishCodeReview.js [hunk 1]
```diff
@@ -0,0 +1,87 @@
...
+
+  ![QR Code](${qrUrl})
+  `;
+
+  fetch(issueUrl)
+    .then(res => res.json())
+    .then(json => {
+      let alreadyCommented = false;
+
+      json.map(comment => {
```
Comment (jaroslav-kubicek) [inline scripts/expo/commands/publishCodeReview.js]: why not find? `json.map(comment => comment.body === body)`

I would probably rename `json` to `comments` as well

File: scripts/expo/commands/publishCodeReview.js [hunk 2]
```diff
@@ -0,0 +1,87 @@
...
+    if (publishError) {
+      throw new Error(publishError);
+    } else if (githubPullRequestId > 0) {
+      // comments only when the PR is opened (but publish always)
+      commentOnGitHub(slug, githubPullRequestId);
+    }
+  });
+};
+
+function commentOnGitHub(buildName, githubPullRequestId) {
```
Comment (jaroslav-kubicek) [inline scripts/expo/commands/publishCodeReview.js]: I would move fn definition above the exported function, one does not simply find exported function in the middle of file

File: scripts/expo/commands/publishCodeReview.js [hunk 3]
```diff
@@ -0,0 +1,87 @@
...
+      });
+
+      if (!alreadyCommented) {
+        fetch(issueUrl, {
+          method: 'POST',
+          headers: { 'User-Agent': 'ci' },
+          body: JSON.stringify({ body }),
+        })
+          .then(res => res.json())
+          .then(json => console.log('GitHub comment created.'));
```
Comment (jaroslav-kubicek) [inline scripts/expo/commands/publishCodeReview.js]: some catch with `console.log` might be useful as well

**Comment by mrtnzlml** [comment]
Thanks @jaroslav-kubicek for the comments. I need to fiddle with the script a little bit more. Now it's important it works... 🙂 I will open another PR with and I will fix there all your comments...

**Comment by jaroslav-kubicek** [comment]
Great, in that case async/await might come handy as well :)

... or eslint rule [no-use-before-define](https://eslint.org/docs/rules/no-use-before-define)

File: scripts/expo/commands/publishCodeReview.js (hunk 3 above)
Comment (mrtnzlml) [reply scripts/expo/commands/publishCodeReview.js]: Can I just let it fail? 😊 It's script in CI anyway so if it doesn't work we need to know (?).

**Pull request merged**